CHANGES.md

```diff
@@ -54,6 +54,11 @@ To be released.
     calling `Swarm<T>.BootstrapAsync()`, `Swarm<T>.PreloadAsync()` and
     `Swarm<T>.StartAsync()`.  [[#353]]
  -  Added `Swarm<T>.BootstrapAsync()` method to connect with seed peers.  [[#353]]
+ -  Added `Block<T>.OrderedTransactions` property to provide transactions ordered in
+    the xor computation of `Block<T>.Id` and `Transaction<T>.Id`.  [[#244], [#355]]
```

Comment:
```suggestion
    a deterministic but unpredictable order.  The order is determined by
    both a `Block<T>.Hash` and a `Transaction<T>.Id`, so that signers cannot
    predict the order of transactions in a block before it's mined.
    [[#244], [#355]]
```